drivers/display/display_sdl.c

```diff
@@ -169,6 +171,9 @@ static int sdl_display_init(const struct device *dev)
 			K_KERNEL_STACK_SIZEOF(disp_data->sdl_thread_stack),
 			sdl_task_thread, (void *)dev, NULL, NULL,
 			CONFIG_SDL_DISPLAY_THREAD_PRIORITY, 0, K_NO_WAIT);
+	k_yield();
```

Comment:
Is this yield still needed if you wait for the semaphore?

Reply:
Fair point, not needed anymore.